ui-grid/ui-grid.d.ts

```diff
@@ -3766,6 +3766,14 @@ declare module uiGrid {
          * You may specify one of the sortingAlgorithms found in the rowSorter service.
          */
         sortCellFiltered?: boolean;
+        /**
+		* (optional) An array of sort directions, specifying the order that they should cycle through as 
```

Comment:
Also did you mean to have the type be UI grid constants and not restrict it to the specific constants that are valid properties